c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -68,6 +89,47 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: template});
 };
 
+
+/**
+ * @param {Date} dateStart
+ * @param {Date} dateEnd
+ * @suppress {checkTypes} it told (found scope | null, required -> scope) at compile()(this.scope)...
+ * @export
+ */
+app.ViewDetailsController.prototype.displayDate = function(dateStart, dateEnd) {
+
+  //angular executes this function so many times it tells 'Error: 10 $digest() iterations reached. Aborting' so
+  // with this quick n dirty method there is no error. Any idea how to do it in the pro way ?
```

Comment:
Actually I think this part could be done in Mako and that we don't need angular (except for translating the dates using "amDateFormat" filters).
I'm currently trying to port the function in a Mako def.